packages/front/src/components/Calendar/component.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+
+import 'react-calendar/dist/Calendar.css';
+import Calendar from 'react-calendar';
+
+import LoopIcon from '../../Icons/Vector.js';
+import { CalendarContainer, IconWrapper } from './styles';
+
+export const CalendarInput = () => {
+  const months = [
+    'Styczeń',
+    'Luty',
+    'Marzec',
+    'Kwiecień',
+    'Maj',
+    'Czerwiec',
+    'Lipiec',
+    'Sierpień',
+    'Wrzesień',
+    'Październik',
+    'Listopad',
+    'Grudzień',
+  ];
+  const [value, onChange] = useState(new Date());
```

Comment:
this should probably be in the interface as currently theres no way of getting information out of the component